Cargo.toml

```diff
@@ -35,3 +35,4 @@ serde = "1.0.197"
 serde_json = "1.0.114"
 thiserror = "1.0.57"
 tokio = { version = "1.36.0", features = ["rt-multi-thread", "sync"] }
+time = "0.3.36"
```

Comment:
Set this to "=0.3.36". The equal ensures the version is actually 0.3.36, not a version that's semantically compatible with 0.3.36

Reply:
Done!